index.js

```diff
@@ -53,3 +53,13 @@ module.exports.butCanIHaveItForDinner =
       "and as a snack... and pretty much whenever you want... " +
       "I guess that includes dinner too."
   }
+module.exports.canIHaveItWithMilk = function getMilkCombinations() {
+  return [
+    "2% milk",
+    "Whole milk",
+    "Almond milk",
+    "Coconut milk",
+    "Soy Milk",
+    "Cinnamon Milk"
+  ];
+}
```

Comment:
Thanks, this is almost there!! I'm a stickler for literals, so can we try a syntax that's more like...

```js
module.exports.whichMilkCanIHaveItWith =
  function* getMilkCombinations() {
    yield "with 2% milk"
    yield "with whole milk"
    yield "with almond milk"
    yield "with coconut milk"
    yield "with soy milk"
    yield "with cinnamon milk"
    yield "with your favorite milk"
    yield "or even dry"
  }
```

Maybe we should add an example in the [examples](https://github.com/towerofnix/why-do-kids-love-the-taste-of-cinnamon-toast-crunch/tree/mistress/examples) folder too, something like this:

```js
const getReasonWhyKidsLoveTheTasteOfCinnamonToastCrunch = require(
  'why-do-kids-love-the-taste-of-cinnamon-toast-crunch')

const getMilksICanHaveCinnamonToastCrunchWith =
      getReasonWhyKidsLoveTheTasteOfCinnamonToastCrunch.canIHaveItWithMilk

for (const milkICouldHaveItWith of getMilksICanHaveCinnamonToastCrunchWith()) {
    console.log(`I could have it ${milkICouldHaveItWith}`)
}
```

I don't know what to name this example though LOL. Just make sure you use an underscore (`_`) between each word, like the other files in the examples folder.

Also, please put a line break above the `module.exports` line - you can look at the picture below to see how the code you added doesn't quite fit the same style as the other ones, but that's just a simple fix.

> <img width="759" alt="Screenshot of bottom four functions in this file" src="https://github.com/towerofnix/why-do-kids-love-the-taste-of-cinnamon-toast-crunch/assets/9948030/0f3ca393-463e-4675-9e0f-eb26a0f68f14">